completion/git-machete.completion.bash

```diff
@@ -1,5 +1,9 @@
 #!/usr/bin/env bash
 
+__gitcomp_empty() {
+  COMPREPLY=('')
```

Comment:
Note that `COMPREPLY=('')` is used in a few other places as well... you can replace that calls with `__gitcomp_empty`, but TBH I'd just remove this function and simply use `COMPREPLY=('')` in line 39